src/less/tables.less

```diff
@@ -124,6 +124,10 @@
     }
 }
 
+.table-default__cell--date {
+    white-space: nowrap;
```

Comment:
Add a reverse footnote that this is done because we always want our dates to be horizontally readable.